main.js

```diff
@@ -25,4 +26,160 @@ closeX.addEventListener('click', () => {
+    mobileDescription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe.",
+    desktopDescription: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scramble.",
+    featured_image1: './images/design.jpg',
+    featured_image2: './images/picturePopup.png',
+    technologies: [
+      'CSS',
+      'html',
+      'bootstrap',
+      'ruby',
+    ],
+    technologies2: [
+      'html',
+      'Ruby on rails',
+      'CSS',
+      'Github',
+    ],
+    live: '#',
+    source: '#',
+  },
+];
+
+const workStory = document.querySelector('.work');
+let lists = '';
+
+stories[0].technologies.forEach((technology) => {
+  lists += `<li>${technology}</li>`;
+});
+for (let i = 0; i < stories.length; i += 1) {
+  workStory.innerHTML += `
+      <div class="story">
+          <img class="picture" src=${stories[i].featured_image1} alt="image about my story"/>
+          <div class="pictureDescription">
+          <h3 class="post">${stories[i].name}</h3>
+          <p class="postDescription">${stories[i].description}</p>
+          <ul class="code">${lists}
+          </ul>
+          <button data-id="${i}" class="project" type="button">See Project</button>
+        </div>
+      </div>
+      `;
+}
+
+let lists2 = '';
+
+stories[0].technologies2.forEach((technology) => {
+  lists2 += `<li>${technology}</li>`;
+});
+
+const modal = document.querySelector('.modal');
+const bodyBlur = document.querySelector('.container');
+const openModal = document.querySelectorAll('.project');
+
+openModal.forEach((n) => n.addEventListener('click', () => {
+  bodyBlur.classList.toggle('active');
+  modal.classList.toggle('active');
+  modal.innerHTML = `
+  <div class="popupWindow">
+  <div class="storyPop">
```

Comment:
- Good job adding the pop up modal 💪🏿 
- [ ]  Kindly make the the background not scrollable and make the modal scrollable so that all the information in the modal can be seen
- [ ] Please check video  screenshot below:

https://user-images.githubusercontent.com/69638013/162085413-accda8fa-8494-409f-86fe-8ee9e9c75be4.mp4

- Please fix for both mobile and desktop 👍🏿

Reply:
ok, thank you